apps/web/lib/team/[slug]/[type]/getServerSideProps.tsx

```diff
@@ -72,6 +72,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     eventSlug: meetingSlug,
     isTeamEvent: true,
     org,
+    fromRedirectOfNonOrgLink: context.query.orgRedirection === "true",
```

Comment:
Convey that event request has been sent from redirected page of non-org link i.e. app.cal.com/teampro redirected to example.cal.com/teampro and event request is sent from that page.